packages/rum-core/src/domain/getSelectorFromElement.ts

```diff
@@ -92,9 +93,10 @@ function getClassSelector(element: Element): string | undefined {
   if (element.tagName === 'BODY') {
     return
   }
-  if (element.classList.length > 0) {
-    for (let i = 0; i < element.classList.length; i += 1) {
-      const className = element.classList[i]
+  const classList = getClassList(element)
+  if (classList.length > 0) {
```

Comment:
**🥜 nitpick:** The `classList.length > 0` condition is useless here, since it's already done in the `for` loop right after. I know it's not your PR that added it, but maybe you can clean it up?

Reply:
Yep nice :)